src/util/styles/gutters.ts

```diff
@@ -1,12 +1,44 @@
 /*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
 import { GutterSize } from '../../components/FixedGrid/types';
 
+export type CSSLengthUnit = string;
+
+export const gutterUnit = {
+  px: 4,
+  rem: 0.4,
+};
+
+const getRemGutterValue = (gutterUnits: number) => {
+  const units = (gutterUnits * gutterUnit.rem).toFixed(1);
```

Comment:
Was the one decimal place chosen intentionally, or more like "it'll do"? I was wondering how much error we could introduce with the lack of a second decimal when talking about bigger gutters...

Reply:
it was intentional, in our font size strategy sets 1rem to 10px. so the one decimal place on a rem is equal to 1 pixel